machine_translation/prepare_data.py

```diff
@@ -160,7 +161,7 @@ def merge_parallel(src_filename, trg_filename, merged_filename):
                 while True:
                     lline = left.readline()
                     rline = right.readline()
-                    if (lline == '') or (rline == ''):
+                    if (lline == '') and (rline == ''):
```

Comment:
But if one of the translations is empty, shouldn't we skip this sentence pair? And how could this cause crashes?

We are now experiencing crashes similar to those reported by the users and trying to find what's going on.